src/opt/ReplaceStdlibShiftPass.cpp

```diff
@@ -29,12 +29,16 @@ bool ReplaceStdlibShiftPass::replaceStdlibShiftBuiltin(llvm::BasicBlock &bb) {
 
     llvm::Module *M = bb.getModule();
     llvm::Function *shifts[6];
-    shifts[0] = M->getFunction("shift___vytuni");
-    shifts[1] = M->getFunction("shift___vysuni");
-    shifts[2] = M->getFunction("shift___vyiuni");
-    shifts[3] = M->getFunction("shift___vyIuni");
-    shifts[4] = M->getFunction("shift___vyfuni");
-    shifts[5] = M->getFunction("shift___vyduni");
+    std::string targetSuffix = std::string("_") + g->target->GetISAString();
+    if (g->singleTargetCompilation) {
+        targetSuffix = "";
+    }
```

Comment:
This is very tiny code but I think it makes sense to extract it as separate function in builtins.h/builtins.cpp. We may reuse it later in multiple places where we determine the target-related suffix (headers generation, function mangling ....).

Reply:
Done